layouts/partials/favicons.html

```diff
@@ -0,0 +1,12 @@
+{{/* Docsy-delta */ -}}
```

Comment:
Let's keep a note about how these were generated:

```suggestion
{{/* Docsy-delta

Icons generated by https://cthedot.de/icongen
from https://github.com/cncf/artwork/blob/main/projects/in-toto/icon/color/in-toto-icon-color.svg
*/ -}}
```